bramkouwen/Music-Mixer
# Adding image

## What does the PR do?
* It adds music-mixer-background.svg to the images folder 

## How do you check this PR?
* Pull down the branch
*  See added file 

## Screenshots 
None.
